packages/manager/modules/pci/src/projects/project/kubernetes/translations/Messages_fr_FR.json

```diff
@@ -9,7 +9,8 @@
   "kube_common_save": "Valider",
   "kube_common_cancel": "Annuler",
   "kube_common_delete": "Supprimer",
-  "kube_common_node_pool_title": "Configurez votre pool de nœuds",
+  "kube_common_node_pool_title": "Configurez votre type de nœuds",
```

Comment:
Is this change legit ? You're keeping `pool de noeuds` just below

Reply:
Yes it's normal.
Because i made some refactoring on different stepper section.
So the initial step can't be stay with same title.

Done with UX.